lib/joi/joi.bundle.js

```diff
@@ -1630,7 +1630,7 @@ module.exports = internals.Any = class {
         }
 
         if (this._invalids.has(value, state, options, this._flags.insensitive)) {
-            errors.push(this.createError(value === '' ? 'any.empty' : 'any.invalid', null, state, options));
+            errors.push(this.createError(value === '' ? 'any.empty' : 'any.invalid', { value, invalids: this._invalids.values({ stripUndefined: true }) }, state, options));
```

Comment:
Generated according to the instructions at [`lib/joi/README.md`](https://github.com/Kashoo/synctos/blob/master/lib/joi/README.md).